diff --git a/src/Entity/Access/GroupRelationshipAccessControlHandler.php b/src/Entity/Access/GroupRelationshipAccessControlHandler.php
index b3e0730..b8193b9 100644
--- a/src/Entity/Access/GroupRelationshipAccessControlHandler.php
+++ b/src/Entity/Access/GroupRelationshipAccessControlHandler.php
@@ -2,6 +2,7 @@
 
 namespace Drupal\group\Entity\Access;
 
+use Drupal\Component\Serialization\Json;
 use Drupal\Core\Entity\EntityAccessControlHandler;
 use Drupal\Core\Entity\EntityHandlerInterface;
 use Drupal\Core\Entity\EntityInterface;
@@ -58,6 +59,24 @@ class GroupRelationshipAccessControlHandler extends EntityAccessControlHandler i
     $relationship_type = $this->entityTypeManager->getStorage('group_content_type')->load($entity_bundle);
     assert($relationship_type instanceof GroupRelationshipTypeInterface);
     $access_control = $this->groupRelationTypeManager->getAccessControlHandler($relationship_type->getPluginId());
+
+    if (!isset($context['group'])) {
+      $request = \Drupal::service('request_stack')->getCurrentRequest();
+      if ($request->getRequestFormat() === 'api_json') {
+        $body = Json::decode($request->getContent());
+        $guuid = $body['data']['relationships']['gid']['data']['id'];
+        if (!empty($guuid)) {
+          $group = \Drupal::entityTypeManager()
+            ->getStorage('group')
+            ->loadByProperties(['uuid' => $guuid]);
+
+          if (!empty($group)) {
+            $context['group'] = reset($group);
+          }
+        }
+      }
+    }
+
     return $access_control->relationshipCreateAccess($context['group'], $account, TRUE);
   }
 
