From 0a8adee789fddfb1f9f9f0d9d54ad1d0d0f25e6d Mon Sep 17 00:00:00 2001
From: Matthew Brabham <brabham.matthew@gmail.com>
Date: Tue, 16 Sep 2025 15:21:37 -0700
Subject: [PATCH] Add maxlength attribute to 'destination' text field in
 ShurlyEditForm

---
 src/Form/ShurlyEditForm.php | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Form/ShurlyEditForm.php b/src/Form/ShurlyEditForm.php
index 6c3148e..fd2af47 100644
--- a/src/Form/ShurlyEditForm.php
+++ b/src/Form/ShurlyEditForm.php
@@ -109,6 +109,7 @@ class ShurlyEditForm extends FormBase {
       '#type' => 'textfield',
       '#title' => 'destination',
       '#value' => $shurly_link[$rid]->destination,
+      '#maxlength' => 2048,
     ];
 
     $form['rid'] = [
-- 
GitLab

