diff --git a/.htaccess b/.htaccess
index 4031da475cf..343192d41cf 100644
--- a/.htaccess
+++ b/.htaccess
@@ -22,6 +22,9 @@ DirectoryIndex index.php index.html index.htm
 AddType image/svg+xml svg svgz
 AddEncoding gzip svgz
 
+# Add webp MIME type so that Apache knows how to handle WebP images
+AddType image/webp .webp
+
 # Most of the following PHP settings cannot be changed at runtime. See
 # sites/default/default.settings.php and
 # Drupal\Core\DrupalKernel::bootEnvironment() for settings that can be
@@ -113,6 +116,310 @@ AddEncoding gzip svgz
   # uncomment the following line:
   # RewriteBase /
 
+  # use a noindex robots on any env not prod
+  RewriteCond %{ENV:AH_SITE_ENVIRONMENT} !prod
+  RewriteRule ^robots\.txt$ robots_noindex.txt [L]
+
+  # Handle autodiscover.xml posts
+  RewriteCond %{REQUEST_URI} ^/autodiscover/autodiscover.xml [NC]
+  RewriteRule .* - [F,L]
+
+  # Blocking bot for user agent.
+  RewriteCond %{HTTP_USER_AGENT} ClaudeBot [NC]
+  RewriteRule .* - [F,L]
+
+  # Redirect all www sites to non www
+  RewriteCond %{HTTP_HOST} ^www\.(.*)$ [NC]
+  RewriteRule ^(.*)$ https://%1/$1 [R=permanent,L]
+
+  # Site Redirects
+  # IS site redirects
+  RewriteCond %{HTTP_HOST} ^(box|banner|zoom)\.oregonstate\.edu [NC]
+  RewriteRule ^ https://technology.oregonstate.edu/services/%1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^is\.oregonstate\.edu [NC]
+  RewriteCond %{REQUEST_URI} /administrative-computing-(0|1) [NC]
+  RewriteRule ^ https://technology.oregonstate.edu/services/banner [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^is\.oregonstate\.edu [NC]
+  RewriteCond %{REQUEST_URI} /zoom [NC]
+  RewriteRule ^ https://technology.oregonstate.edu/services/zoom [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^is\.oregonstate\.edu [NC]
+  RewriteCond %{REQUEST_URI} /canvas [NC]
+  RewriteRule ^ https://learn.oregonstate.edu/canvas-visitor-login [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^is\.oregonstate\.edu [NC]
+  RewriteCond %{REQUEST_URI} /duo [NC]
+  RewriteRule ^ https://technology.oregonstate.edu/services/duo [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^is\.oregonstate\.edu [NC]
+  RewriteCond %{REQUEST_URI} /telecom/?$ [NC]
+  RewriteRule ^ https://technology.oregonstate.edu/services/phones-and-voicemail [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^is\.oregonstate\.edu [NC]
+  RewriteCond %{REQUEST_URI} /exchangeonline [NC]
+  RewriteRule ^ https://technology.oregonstate.edu/services/outlook-exchange-online [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^is\.oregonstate\.edu [NC]
+  RewriteCond %{REQUEST_URI} /cts [NC,OR]
+  RewriteCond %{REQUEST_URI} /media-services [NC]
+  RewriteRule ^ https://uit.oregonstate.edu/tsa/academic-technologies [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^is\.oregonstate\.edu [NC]
+  RewriteCond %{REQUEST_URI} /telephone-coordinators [NC]
+  RewriteRule ^ https://mysupport.oregonstate.edu/kb?id=kb_article_view&sysparm_article=KB0010941 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^is\.oregonstate\.edu [NC]
+  RewriteCond %{REQUEST_URI} /telecom/request-phonenetwork-service-or-repair [NC]
+  RewriteRule ^ https://mysupport.oregonstate.edu/esp?id=sc_cat_item&sys_id=633c95fa97ed8ed01b28bf98c253afa1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^is\.oregonstate\.edu [NC]
+  RewriteCond %{REQUEST_URI} /ois [NC]
+  RewriteRule ^ https://uit.oregonstate.edu/ois [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^is\.oregonstate\.edu [NC]
+  RewriteRule ^ https://uit.oregonstate.edu [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^linc\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://uit.oregonstate.edu/linc [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^uit-at\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://uit.oregonstate.edu/tsa/academic-technologies [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^iar\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://data.oregonstate.edu [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^apa\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://curriculummanagement.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^apa\.stage\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://curriculummanagement.stage.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^apa\.dev\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://curriculummanagement.dev.oregonstate.edu/$1 [R=permanent,L]
+
+  # Search Advocate Site Redirection
+  RewriteCond %{HTTP_HOST} ^searchadvocate\.oregonstate\.edu [NC]
+  RewriteRule ^about/becoming-search-advocate https://hr.oregonstate.edu/search-advocate-program/becoming-search-advocate [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^searchadvocate\.oregonstate\.edu [NC]
+  RewriteRule ^search-advocate-resources https://hr.oregonstate.edu/search-advocate-program/search-advocate-resources [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^searchadvocate\.oregonstate\.edu [NC]
+  RewriteRule ^home/search-advocate-workshops https://hr.oregonstate.edu/search-advocate-program/search-advocate-workshops [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^searchadvocate\.oregonstate\.edu [NC]
+  RewriteRule ^about/osu-search-advocate-directory/program-staff https://hr.oregonstate.edu/search-advocate-program/search-advocate-program-staff [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^searchadvocate\.oregonstate\.edu [NC]
+  RewriteRule ^about/search-advocacy-beyond-osu https://hr.oregonstate.edu/search-advocate-program/search-advocacy-beyond-osu [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^searchadvocate\.oregonstate\.edu [NC]
+  RewriteRule ^find-search-advocate https://hr.oregonstate.edu/search-advocate-program/find-search-advocate [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^searchadvocate\.oregonstate\.edu [NC]
+  RewriteRule ^ https://hr.oregonstate.edu/search-advocate-program [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^cimrs\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://cimers.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^covid\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://studenthealth.oregonstate.edu/covid-19-info [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^dce\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://see.oregonstate.edu/dce/ [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^dosa\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://studentaffairs.oregonstate.edu/DOSA [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^dpd\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://dpo.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^international\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://global.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^nextgreatstartup\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://business.oregonstate.edu/nextgreatstartup [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^semiconductor\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://engineering.oregonstate.edu/research/semiconductor/ [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^today\.oregonstate\.edu$ [NC]
+  RewriteRule ^news/?$ https://news.oregonstate.edu/ [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^today\.oregonstate\.edu [NC]
+  RewriteRule ^news/(.*)$ https://news.oregonstate.edu/news/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^robotics\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://engineering.oregonstate.edu/CoRIS [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^transboundarywaters\.science\.oregonstate\.edu [NC,OR]
+  RewriteCond %{HTTP_HOST} ^transboundarywaters\.orst.\edu [NC]
+  RewriteRule ^(.*)$ https://transboundarywaters.ceoas.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^tourism\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://seagrant.oregonstate.edu/tourism/ [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^ela\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://see.oregonstate.edu/ela/ [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^craftcenter\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://see.oregonstate.edu/craft-center/ [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^osupc\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://see.oregonstate.edu/ela/osu-program-council/ [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^omn\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://see.oregonstate.edu/ela/orange-media-network/ [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^sccs\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://scs.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^dri\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://arcs.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^healthycampus\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://beaverhealthy.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^undocumented\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://immigrationbeavs.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^facultyaffairs\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://academicaffairs.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^accreditation\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://academicaffairs.oregonstate.edu/support/accreditation [R=permanent,L]
+
+  # Aye yi yi FA => DFA Redirects
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/budget/tuition-fees$
+  RewriteRule ^(.*)$ https://budget.oregonstate.edu/tuition-fees [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/classification-specifications$
+  RewriteRule ^(.*)$ https://hr.oregonstate.edu/classification-specifications [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/controllers-unit$
+  RewriteRule ^(.*)$ https://finance.oregonstate.edu/ [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/controllers-unit/business-process-forum$
+  RewriteRule ^(.*)$ https://finance.oregonstate.edu/business-process-forum [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/controllers-unit/payroll$
+  RewriteRule ^(.*)$ https://finance.oregonstate.edu/payroll [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/controllers-unit/payroll/direct-deposit-sign-up$
+  RewriteRule ^(.*)$ https://finance.oregonstate.edu/payroll/direct-deposit-how-sign [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/controllers-unit/payroll/w2-w4-tax-information$
+  RewriteRule ^(.*)$ https://finance.oregonstate.edu/w2-w4-tax-information [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/controllers-unit/student-accounts$
+  RewriteRule ^(.*)$ https://finance.oregonstate.edu/services/student-accounts [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/controllers-unit/student-accounts/1098t$
+  RewriteRule ^(.*)$ https://finance.oregonstate.edu/1098t-online-access [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/controllers-unit/student-accounts/billing$
+  RewriteRule ^(.*)$ https://finance.oregonstate.edu/student-accounts/student-billing [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/controllers-unit/student-accounts/billing/payment-plans$
+  RewriteRule ^(.*)$ https://finance.oregonstate.edu/student-accounts/payment-plans [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/controllers-unit/student-accounts/tuition-refund-appeal$
+  RewriteRule ^(.*)$ https://finance.oregonstate.edu/student-accounts/tuition-refund-appeal [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/controllers-unit/student-resources/information-regarding-personal-checks$
+  RewriteRule ^(.*)$ https://finance.oregonstate.edu/information-regarding-personal-checks [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/controllers-unit/travel-and-expense$
+  RewriteRule ^(.*)$ https://finance.oregonstate.edu/accounts-payable [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/eProcurement$
+  RewriteRule ^(.*)$ https://eprocurement.oregonstate.edu/ [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/faq/what-are-different-types-mandatory-fees$
+  RewriteRule ^(.*)$ https://finance.oregonstate.edu/what-are-different-types-mandatory-fees [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/fis-manual/000-introductory-material/003-list-exhibits/account-codes/group$
+  RewriteRule ^(.*)$ https://oregonstateuniversity.sharepoint.com/sites/CUPP [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/fiscal-program/guidelines/exhibits/ex003-02-account-codes$
+  RewriteRule ^(.*)$ https://oregonstateuniversity.sharepoint.com/sites/CUPP [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/surplus/public-sales/online-sales$
+  RewriteRule ^(.*)$ https://surplus.oregonstate.edu/public-sales/online-sales [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/fiscal-policy-program/03-110-206-parking-fines-and-permit-sales$
+  RewriteRule ^(.*)$ https://oregonstateuniversity.sharepoint.com/sites/CUPP/SitePages/Parking-Fines-and-Permit-Sales.aspx [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^fa\.oregonstate\.edu$ [NC]
+  RewriteCond %{REQUEST_URI} ^/$
+  RewriteRule ^(.*)$ http://dfa.oregonstate.edu/ [R=permanent,L]
+  # End FA => DFA Redirects
+
+  RewriteCond %{HTTP_HOST} ^water\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://inr.oregonstate.edu/institute-water-watersheds [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^mycard\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://idx.transactcampus.com/accounts/oregonstate-edu/osu-id-card [R=permanent,L]
+
+  # SF Lightning web compontents don't use full paths for assets.
+  # See https://salesforce.stackexchange.com/questions/211581/lightning-icons-svg-not-displaying-in-lightning-out
+  RewriteCond %{HTTP_HOST} hr\.oregonstate\.edu [NC]
+  RewriteCond %{REQUEST_URI} /_slds/icons/utility-sprite/svg [NC]
+  RewriteRule ^(.*)$ https://oregonstate.my.site.com%{REQUEST_URI} [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} hr\.oregonstate\.edu [NC]
+  RewriteRule sites/hr.oregonstate.edu/files/documents/general/incl_weather_matrix.pdf https://hr.oregonstate.edu/closures-or-curtailment-operations-including-inclement-weather [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} welcome\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://newstudents.oregonstate.edu/ [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} trio-sss\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://trio.oregonstate.edu/ [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} gradschool\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://graduate.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^gradschool\.stage\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://graduate.stage.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^gradschool\.dev\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://graduate.dev.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ufio\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://fpm.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} thinkdifferently\.liberalarts\.oregonstate\.edu [NC]
+  RewriteRule ^(.*)$ https://liberalarts.oregonstate.edu/cla-contact-form [R=permanent,L]
+
+  # Redirects for SSL of the site
+  RewriteCond %{HTTPS} off
+  RewriteCond %{HTTP:X-Forwarded-Proto} !https
+  RewriteCond %{HTTP_HOST} !(.*)\.?localhost [NC]
+  RewriteCond %{HTTP_HOST} !^osuwams\.ews\.oregonstate\.edu$ [NC]
+  RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [NE,L,R=permanent]
+
   # Redirect common PHP files to their new locations.
   RewriteCond %{REQUEST_URI} ^(.*)?/(install\.php) [OR]
   RewriteCond %{REQUEST_URI} ^(.*)?/(rebuild\.php)
@@ -184,3 +491,9 @@ AddEncoding gzip svgz
   # Disable Proxy header, since it's an attack vector.
   RequestHeader unset Proxy
 </IfModule>
+<IfModule mod_setenvif.c>
+  #SetEnvIf AH_CLIENT_IP ^23\.108\.252\.102$ DENY=1
+  Order allow,deny
+  Allow From All
+  Deny from env=DENY
+</IfModule>
