diff --git a/.htaccess b/.htaccess
index 1575c51f3d..33003df0f2 100644
--- a/.htaccess
+++ b/.htaccess
@@ -22,6 +22,9 @@ DirectoryIndex index.php index.html index.htm
 AddType image/svg+xml svg svgz
 AddEncoding gzip svgz
 
+# Add webp MIME type so that Apache knows how to handle WebP images
+AddType image/webp .webp
+
 # Most of the following PHP settings cannot be changed at runtime. See
 # sites/default/default.settings.php and
 # Drupal\Core\DrupalKernel::bootEnvironment() for settings that can be
@@ -120,6 +123,24 @@ AddEncoding gzip svgz
   # uncomment the following line:
   # RewriteBase /
 
+  # use a noindex robots on any env not prod
+  RewriteCond %{ENV:AH_SITE_ENVIRONMENT} !prod
+  RewriteRule ^robots\.txt$ robots_noindex.txt [L]
+
+  # Site Redirects
+  RewriteCond %{HTTP_HOST} ^prax\.oregonstate\.edu [NC]
+  RewriteRule (.*) https://liberalarts.oregonstate.edu/prax/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^dpd\.oregonstate\.edu [NC]
+  RewriteRule (.*) https://dpo.oregonstate.edu/$1 [R=permanent,L]
+
+  # Redirects for SSL of the site
+  RewriteCond %{HTTPS} off
+  RewriteCond %{HTTP:X-Forwarded-Proto} !https
+  RewriteCond %{HTTP_HOST} !(.*)\.?localhost [NC]
+  RewriteCond %{HTTP_HOST} !^osuwams\.ews\.oregonstate\.edu$ [NC]
+  RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [NE,L,R=301]
+
   # Redirect common PHP files to their new locations.
   RewriteCond %{REQUEST_URI} ^(.*)?/(install\.php) [OR]
   RewriteCond %{REQUEST_URI} ^(.*)?/(rebuild\.php)
