diff --git a/.htaccess b/.htaccess
index 4031da475c..484db3366b 100644
--- a/.htaccess
+++ b/.htaccess
@@ -22,6 +22,9 @@ DirectoryIndex index.php index.html index.htm
 AddType image/svg+xml svg svgz
 AddEncoding gzip svgz
 
+# Add webp MIME type so that Apache knows how to handle WebP images
+AddType image/webp .webp
+
 # Most of the following PHP settings cannot be changed at runtime. See
 # sites/default/default.settings.php and
 # Drupal\Core\DrupalKernel::bootEnvironment() for settings that can be
@@ -113,6 +116,64 @@ AddEncoding gzip svgz
   # uncomment the following line:
   # RewriteBase /
 
+  # use a noindex robots on any env not prod
+  RewriteCond %{ENV:AH_SITE_ENVIRONMENT} !prod
+  RewriteRule ^robots\.txt$ robots_noindex.txt [L]
+
+  # Handle autodiscover.xml posts
+  RewriteCond %{REQUEST_URI} ^/autodiscover/autodiscover.xml [NC]
+  RewriteRule .* - [F,L]
+
+  # Blocking bot for user agent.
+  RewriteCond %{HTTP_USER_AGENT} ClaudeBot [NC]
+  RewriteRule .* - [F,L]
+
+  # Redirect all www sites to non www
+  RewriteCond %{HTTP_HOST} ^www\.(.*)$ [NC]
+  RewriteRule ^(.*)$ https://%1/$1 [R=permanent,L]
+
+  # Site Redirects
+  RewriteCond %{HTTP_HOST} ^box\.oregonstate\.edu [NC]
+  RewriteRule (.*) https://uit.oregonstate.edu/box [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^cimrs\.oregonstate\.edu [NC]
+  RewriteRule (.*) https://cimers.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^covid\.oregonstate\.edu [NC]
+  RewriteRule (.*) https://studenthealth.oregonstate.edu/covid-19-info [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^dosa\.oregonstate\.edu [NC]
+  RewriteRule (.*) https://studentaffairs.oregonstate.edu/DOSA [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^dpd\.oregonstate\.edu [NC]
+  RewriteRule (.*) https://dpo.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^international\.oregonstate\.edu [NC]
+  RewriteRule (.*) https://global.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^nextgreatstartup\.oregonstate\.edu [NC]
+  RewriteRule (.*) https://business.oregonstate.edu/nextgreatstartup [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^robotics\.oregonstate\.edu [NC]
+  RewriteRule (.*) https://engineering.oregonstate.edu/CoRIS [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^semiconductor\.oregonstate\.edu [NC]
+  RewriteRule (.*) https://engineering.oregonstate.edu/research/semiconductor/ [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^transboundarywaters\.science\.oregonstate\.edu [NC]
+  RewriteRule (.*) https://transboundarywaters.ceoas.oregonstate.edu/$1 [R=permanent,L]
+
+  RewriteCond %{HTTP_HOST} ^www.transboundarywaters\.orst.\edu [NC]
+  RewriteRule (.*) https://transboundarywaters.ceoas.oregonstate.edu/$1 [R=permanent,L]
+
+
+  # Redirects for SSL of the site
+  RewriteCond %{HTTPS} off
+  RewriteCond %{HTTP:X-Forwarded-Proto} !https
+  RewriteCond %{HTTP_HOST} !(.*)\.?localhost [NC]
+  RewriteCond %{HTTP_HOST} !^osuwams\.ews\.oregonstate\.edu$ [NC]
+  RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [NE,L,R=301]
+
   # Redirect common PHP files to their new locations.
   RewriteCond %{REQUEST_URI} ^(.*)?/(install\.php) [OR]
   RewriteCond %{REQUEST_URI} ^(.*)?/(rebuild\.php)
