diff --git a/.htaccess b/.htaccess
index 1575c51f3d..5e440b95d0 100644
--- a/.htaccess
+++ b/.htaccess
@@ -120,6 +120,21 @@ AddEncoding gzip svgz
   # uncomment the following line:
   # RewriteBase /
 
+  # use a noindex robots on any env not prod
+  RewriteCond %{ENV:AH_SITE_ENVIRONMENT} !prod
+  RewriteRule ^robots\.txt$ robots_noindex.txt [L]
+
+  # Site Redirects
+  RewriteCond %{HTTP_HOST} ^prax\.oregonstate\.edu [NC]
+  RewriteRule (.*) https://liberalarts.oregonstate.edu/prax/$1 [R=permanent,L]
+
+  # Redirects for SSL of the site
+  RewriteCond %{HTTPS} off
+  RewriteCond %{HTTP:X-Forwarded-Proto} !https
+  RewriteCond %{HTTP_HOST} !(.*)\.?localhost [NC]
+  RewriteCond %{HTTP_HOST} !^osuwams\.ews\.oregonstate\.edu$ [NC]
+  RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [NE,L,R=301]
+
   # Redirect common PHP files to their new locations.
   RewriteCond %{REQUEST_URI} ^(.*)?/(install\.php) [OR]
   RewriteCond %{REQUEST_URI} ^(.*)?/(rebuild\.php)
