diff --git a/.htaccess b/.htaccess
index 4031da475cf..8862edeca7f 100644
--- a/.htaccess
+++ b/.htaccess
@@ -22,6 +22,16 @@ DirectoryIndex index.php index.html index.htm
 AddType image/svg+xml svg svgz
 AddEncoding gzip svgz
 
+# Add webp MIME type so that Apache knows how to handle WebP images
+AddType image/webp .webp
+
+# Add avif MIME type
+AddType image/avif .avif
+AddType image/avif-sequence .avifs
+
+# Add JPEG-XL MIME type
+AddType image/jxl .jxl
+
 # Most of the following PHP settings cannot be changed at runtime. See
 # sites/default/default.settings.php and
 # Drupal\Core\DrupalKernel::bootEnvironment() for settings that can be
@@ -113,6 +123,40 @@ AddEncoding gzip svgz
   # uncomment the following line:
   # RewriteBase /
 
+  # use a noindex robots on any env not prod
+  RewriteCond %{ENV:AH_SITE_ENVIRONMENT} !prod
+  RewriteRule ^robots\.txt$ robots_noindex.txt [L]
+
+  # Handle autodiscover.xml posts
+  RewriteCond %{REQUEST_URI} ^/autodiscover/autodiscover.xml [NC]
+  RewriteRule .* - [F,L]
+
+  # Redirect all www sites to non www
+  RewriteCond %{HTTP_HOST} ^www\.(.*)$ [NC]
+  RewriteRule ^(.*)$ https://%1/$1 [R=permanent,L]
+
+  # Site Redirects
+  RewriteCond %{HTTP_HOST} technology.oregonstate.edu [NC]
+  RewriteCond %{REQUEST_METHOD} ^OPTIONS$
+  RewriteRule ^canvas-outage(/.*)?$ - [R=405,L]
+
+  # IS site redirects
+  RewriteCond %{HTTP_HOST} ^(banner|zoom)\.oregonstate\.edu [NC]
+  RewriteRule ^ https://technology.oregonstate.edu/services/%1 [R=permanent,L]
+
+  # SF Lightning web compontents don't use full paths for assets.
+  # See https://salesforce.stackexchange.com/questions/211581/lightning-icons-svg-not-displaying-in-lightning-out
+  RewriteCond %{HTTP_HOST} hr\.oregonstate\.edu [NC]
+  RewriteCond %{REQUEST_URI} /_slds/icons/utility-sprite/svg [NC]
+  RewriteRule ^(.*)$ https://oregonstate.my.site.com%{REQUEST_URI} [R=permanent,L]
+
+  # Redirects for SSL of the site
+  RewriteCond %{HTTPS} off
+  RewriteCond %{HTTP:X-Forwarded-Proto} !https
+  RewriteCond %{HTTP_HOST} !(.*)\.?localhost [NC]
+  RewriteCond %{HTTP_HOST} !^osuwams\.ews\.oregonstate\.edu$ [NC]
+  RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [NE,L,R=permanent]
+
   # Redirect common PHP files to their new locations.
   RewriteCond %{REQUEST_URI} ^(.*)?/(install\.php) [OR]
   RewriteCond %{REQUEST_URI} ^(.*)?/(rebuild\.php)
@@ -184,3 +228,9 @@ AddEncoding gzip svgz
   # Disable Proxy header, since it's an attack vector.
   RequestHeader unset Proxy
 </IfModule>
+<IfModule mod_setenvif.c>
+  #SetEnvIf AH_CLIENT_IP ^23\.108\.252\.102$ DENY=1
+  Order allow,deny
+  Allow From All
+  Deny from env=DENY
+</IfModule>
